sample.env

```diff
@@ -0,0 +1,2 @@
+# Change this to a random string if you want to host this
```

Comment:
This file is not for hosting, but only for local project setup, so could you please correct this comment?
You could maybe say `Change this to a random string to run the project locally`.